report_warranty.php

```diff
@@ -388,6 +388,7 @@ function SetLegends($data, $format) {
 // first page - the pie chart
 
 $pdf = new PDF_Diag();
+include_once("loadfonts.php");
```

Comment:
umm... including the same load file twice is just not a real good fix.  If the position of the other load is the issue then just move it up.  This pull, however, is denied.